packages/perseus/src/widgets/expression/expression.tsx

```diff
@@ -745,7 +745,12 @@ const keypadConfigurationForProps = (
         extraKeys = ["PI"];
     }
 
-    return {keypadType, extraKeys, times: widgetOptions.times};
+    return {
+        keypadType,
+        extraKeys,
+        times: widgetOptions.times,
+        scientific: widgetOptions.buttonSets.includes("scientific"),
```

Comment:
I forget, why do we have to add this for the `scientific` button set but not the rest of them? Should we comment why this particular button set is special?

Reply:
I'm having trouble remembering as well 🤔
I tested without this, and it doesn't _seem_ to have an impact, so I'll remove it, but I'm sure if we do end up implementing this, we will quickly figure out why this was needed 😅